docs/topics/rotate-certs.md

```diff
@@ -0,0 +1,127 @@
+# Rotating Kubernetes Certificates
+
+## Prerequisites
+
+This guide assumes that you already have deployed a cluster using `aks-engine` and the cluster is in a healthy state.
+
+## Certificate Rotation
+
+This document provides guidance on how to rotate certificates on an existing AKS Engine cluster and recommendations for adopting `aks-engine rotate-certs` as a tool.
+
+### Know before you go
+
+In order to ensure that your `aks-engine rotate-certs` operation runs smoothly, there are a few things you should be aware of before getting started.
+
+1. You will need access to the API Model (`apimodel.json`) that was generated by `aks-engine deploy` (by default this file is placed into a relative directory that looks like `_output/<clustername>/`).
```

Comment:
can we say "that was generated by `aks-engine deploy` or `aks-engine generate`" ?

Reply:
yes, go crazy with suggestions on how to improve the docs